src/components/question-wrapper.tsx

```diff
@@ -52,33 +52,40 @@ export default class QuestionWrapper extends React.Component<IProps, IState> {
   public render() {
     const { activeTab } = this.state;
     const { authoredState } = this.props;
-    const { teacherTip, exemplar, correctExplanation, distractorsExplanation } = authoredState;
+    const { teacherTip, exemplar, correctExplanation, distractorsExplanation, location } = authoredState;
+
+    const _location = location ? location : QuestionWrapperLocation.Bottom;
 
     let wrapperClass = css.questionWrapper;
     if (this.isInteractive) {
       // Special style, different icons.
       wrapperClass += " " + css.interactiveWrapper;
     }
 
-    let footer = null;
+    let visibleText = null;
     let wrappedContentClass = css.wrappedContent;
     if (activeTab === "Correct") {
-      footer = correctExplanation;
+      visibleText = correctExplanation;
       wrappedContentClass += " " + css.correct;
     } else if (activeTab === "Distractors") {
-      footer = distractorsExplanation;
+      visibleText = distractorsExplanation;
       wrappedContentClass += " " + css.distractors;
     } else if (activeTab === "TeacherTip") {
-      footer = teacherTip;
+      visibleText = teacherTip;
       wrappedContentClass += " " + css.teacherTip;
     } else if (activeTab === "Exemplar") {
-      footer = exemplar;
+      visibleText = exemplar;
       wrappedContentClass += " " + css.examplar;
```

Comment:
I hadn't noticed this before, but the spelling of "examplar" is both odd and inconsistent. It occurs here as well as in it's definition in the question-wrapper.sass file, around line 72. Prolly not worth fixing, but it jumped out at me.

Attach files by dragging & dropping, selecting or pasting them.

Reply:
Ah yes, it looks like there was a typo in the css file.